shell/client/styles/_topbar.scss

```diff
@@ -75,8 +75,7 @@ body>.topbar {
   background-color: $topbar-background-color;
   color: #ccc;
 
-  // Using 0 rather than CSS default ("auto") allows introjs to highlight elements in the topbar.
-  z-index: 0;
+  z-index: 1;
```

Comment:
I'm slightly nervous about this because I think by establishing a separate stacking context it might prevent other modals that aren't direct descendants of the topbar from being able to cover the topbar/sidebar.

Can you double-check that this doesn't inadvertently cover, say, the modal that you'd see at http://local.sandstorm.io:6080/admin/identity when configuring a login provider?  If it does, can you try switching this back to `z-index: 0;` and push the grain-frames further back (potentially negative z-index) instead?